pyprof/prof/tensor.py

```diff
@@ -19,14 +19,14 @@
 
 class Tensor(object):
     def __init__(self, shape, dtype):
-        assert type(shape) == tuple
+        assert type(shape) in [tuple, list]
         assert dtype in Dtype.types()
-        self._shape = shape
+        self._shape = list(shape)
```

Comment:
Can you change the name of `_shape` to indicate that it is a list of shapes. Otherwise, it may be confused later as a single tensor shape, which is how that name has been referred to in the rest of the code base.

Reply:
`_shape` is not a list of shapes. Its just a list of integers e.g. `[4,4]`. The user can, however, pass a tuple or a list e.g. `(4,4)` or `[4,4]` as argument. We just change everything to `[4,4]` for uniformity, hence the syntax `list(shape)`.